tests/integration/test_stages.py

```diff
@@ -1,3 +1,4 @@
+# pylint:disable=protected-access,no-self-use
```

Comment:
@jbzdak as far as I can tell, on master there's only one file-wide `protected-access` in `test_mixins.py`, and it dates back to August, and in August I was much less strict in terms of code quality.

And there're three no-self-use, one file-wide (again in test_mixins). So, no, I'm not convinced, there might be placed where it's justified, but you can suppress it there, locally, for one line. With file-wide suppression you might suppress valid violations as well.